orquesta/tests/unit/composition/native/test_workflow_join.py

```diff
@@ -231,7 +231,8 @@ def test_join_count(self):
                     'barrier': 2
                 },
                 {
-                    'id': 'noop'
+                    'id': 'noop',
+                    'splits': ['noop']
```

Comment:
```suggestion
                    'splits': ['noop'],
```

I'm a fan of ending commas, since it makes diffs one line shorter.

Original code (no ending comma):

```python
{
  "id": "noop"
}
```

New code (no ending comma):

```python
{
  "id": "noop",
  "splits": ["noop"]
}
```

The diff is a little convoluted compared to what was added:

```diff
 {
-  "id": "noop"
+  "id": "noop",
+  "splits": ["noop"]
 }
```

But if you have ending commas, the diff is smaller to review, and is a closer match to what was actually added.

Old code (with ending comma):

```python
{
  "id": "noop",
}
```

New code (same as before, but with ending comma):

```python
{
  "id": "noop",
  "splits": ["noop"],
}
```

New diff:

```diff
 {
   "id": "noop",
+  "splits": ["noop"],
 }
```

But this is also a matter of preference, so feel free to instantly resolve this as well if you disagree.

Reply:
I don't disagree. But we haven't been doing this code style in the project.  So to introduce this in this one line is inconsistent.  I prefer that we fix it across the board in separate PR.